src/components/Receiving/ItemDetails.css

```diff
@@ -0,0 +1,3 @@
+.editableFieldWrapper {
+  margin-top: 14px;
```

Comment:
doesn't it look like a hack to you? Maybe it make sense to check `Layout` helpers https://ux.folio.org/storybook/?selectedKind=Layout&selectedStory=Available%20classes&full=0&addons=1&stories=1&panelRight=0&addonPanel=REACT_STORYBOOK%2Freadme%2Fpanel
or LayoutGrid https://ux.folio.org/storybook/?selectedKind=LayoutGrid&selectedStory=Basic%20Usage&full=0&addons=1&stories=1&panelRight=0&addonPanel=REACT_STORYBOOK%2Freadme%2Fpanel

Reply:
Input(select) has default style `margin-bottom: 1rem`